### ISSUE TITLE
Ghosty Dev Web Projects are duplicated

### ISSUE THREAD

**Issue by adrianmarinwork** [GhostyORG/Ghosty-web#73]
Hello, I don't know if you have done this intended or it's happening because of some issue, but in the Projects page all the projects are duplicated.
The ul list it's being added two times to the div container with the id `orgProjects`.
In the case that it's not done intended I can take care of this if you want.

**Comment by killerclaws12** [comment]
@Abir-Tx respond to this.

**Comment by killerclaws12** [comment]
Hey @adrianmarinwork. I just looked into this. I believe he has to restart the vercel thing like rebuild the project if makes sense, when he does that, that should fix the problem because i also noticed, some of the projects are glitched on the tab. This happened before but always got fixed when he restarted the vercel thingy.

**Comment by Abir-Tx** [comment]
This happened once before iirc. Rebuilding fixed the problem earlier. Anyway, if you can please fix it @adrianmarinwork as I am in a very busy schedule recently. Or you can maybe start a draft PR so that I can also work on it sometimes

Thank you ❤

**Comment by killerclaws12** [comment]
^^

**Comment by adrianmarinwork** [comment]
I'll take care of this and if I mange to fix it I'll create a pull request.

**Comment by Abir-Tx** [comment]
> I'll take care of this and if I mange to fix it I'll create a pull request.

Thank you so much 😍

**Comment by adrianmarinwork** [comment]
Okay, I've created a pull request with the fixed that I have made.
It's based in a React solution, what I mean is that I have moved the logic of fetching the projects from the github repository into a useEffect variable that when the asyn function has finished updates the list generated in the HTML.
To be honest I'm not really experienced with React but thats the best solution that I found. 
The issue was that because the code was in a external javascript and the method was being called in the return template each time the template was rendered the method was called, because of that the list was rendered more than one time (for me locally the list has been added even 4 times).
If you dont like this solution I have an uglier solution in mind but that it should work.
Just let me know!
